auth0_flutter/ios/Classes/WebAuth/WebAuthModels.swift

```diff
@@ -0,0 +1,15 @@
+enum SafariViewControllerProperty: String, CaseIterable {
+    case presentationStyle
+}
+
+struct SafariViewController {
+    var presentationStyle: UIModalPresentationStyle = UIModalPresentationStyle.fullScreen
+    
+    static let key = "safariViewController";
+    
+    init(from dictionary: [String: Any?]) {
+        if let presentationStyle = dictionary[SafariViewControllerProperty.presentationStyle] as? Int {
+            self.presentationStyle = UIModalPresentationStyle(rawValue: presentationStyle) ?? UIModalPresentationStyle.fullScreen
```

Comment:
The default value (`?? UIModalPresentationStyle.fullScreen`) is not necessary here because it's already set in the property (L6).

Reply:
If I don't add this, the compiler complains that I must unwrap it first, looks like `UIModalPresentationStyle(rawValue: presentationStyle)` returns `UIModalPresentationStyle?` (I guess incase the value passed as `rawValue` is invalid).

Reply:
You can add another clause to the `if let` for this. E.g:

```swift
if let presentationStyle = dictionary[SafariViewControllerProperty.presentationStyle] as? Int, let uiModalPresentationStyle = UIModalPresentationStyle(rawValue: presentationStyle) {
    self.presentationStyle = uiModalPresentationStyle
}
```

Reply:
Much better! Thanks 👍🏻